Tasks/AzureAppServiceManageV0/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.218.0",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.219.1",
```

Comment:
Publish azure-pipelines-tasks-azure-arm-rest-v2 and update to 3.221.0

Reply:
Done